### PR TITLE
Blog- rails that scales - powered by vitess

### PR DESCRIPTION
**Pull request by GuptaManan100**
This PR adds the blog `Rails that scales - Powered by Vitess`

### CODE REVIEW

**Comment by GuptaManan100** [comment]
To be merged on 17th Dec 2021

File: content/en/blog/2021-12-17-rails-that-scales.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+author: 'Manan Gupta'
+date: 2021-12-17
+slug: '2021-12-17-rails-that-scales'
+tags: ['Vitess','CNCF', 'Rails', 'scaling','web','framework']
```
Comment (askdba) [inline content/en/blog/2021-12-17-rails-that-scales.md]: Can you add MySQL as tag? This is required for popular RSS readers to catch our blog.

Comment (GuptaManan100) [reply content/en/blog/2021-12-17-rails-that-scales.md]: Yes, sounds good!

**Review by askdba — APPROVED** [review]
LGTM

File: content/en/blog/2021-12-17-rails-that-scales.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+```
+
+Now, we can follow along with the rest of the getting started guide. You can go into the Rails console and run the commands for creating an article, filtering them, viewing them and updating them. They all work just as we’d like them to.
+
+You can look at the examples from the Rails guides and the Vschema that we used to get the optimal performance for them in the [Vitess framework testing repository](https://github.com/planetscale/vitess-framework-testing/tree/main/frameworks/ruby/rails6/rails-guide). 
+
+We cover all the examples from the five guides: [Active Records Migration](https://guides.rubyonrails.org/active_record_migrations.html), [Active Records Validations](https://guides.rubyonrails.org/active_record_validations.html), [Active Records Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), [Active Records Associations](https://guides.rubyonrails.org/association_basics.html) and [Active Records Query Interface](https://guides.rubyonrails.org/active_record_querying.html).
+
+## Trainwrecks Along the way
```
Comment (vmg) [inline content/en/blog/2021-12-17-rails-that-scales.md]: I don't think calling these "trainwrecks" is a good idea. How are they trainwrecks anyway?

Comment (vmg) [reply content/en/blog/2021-12-17-rails-that-scales.md]: Oh, I now get it's a Rails pun. :sweat_smile:

I would recommend changing it anyway, trainwreck sounds like something gone badly, and all those are issues that got resolved quite easily.

Comment (GuptaManan100) [reply content/en/blog/2021-12-17-rails-that-scales.md]: Do you have any suggestions in mind? How do you feel about `roadblocks`?

Comment (askdba) [reply content/en/blog/2021-12-17-rails-that-scales.md]: `roadblocks` or ` speedbumps` are better if they were resolved quite easily. I agree with @vmg that `trainwrecks` are usually referred to as disasters.

Comment (GuptaManan100) [reply content/en/blog/2021-12-17-rails-that-scales.md]: @vmg @askdba I like both `roadblocks` and `speedbumps`. Which one do you prefer?

Comment (GuptaManan100) [reply content/en/blog/2021-12-17-rails-that-scales.md]: Going ahead with `speedbumps`, we can change it if we decide otherwise

**Comment by GuptaManan100** [comment]
Changing the release date to be 16 Dec, so that it is published before the monthly meeting call and we can add it there

**Pull request merged**